File: sample.rb
#!ruby
#coding:utf-8

CMD = ".\\iwm_DirFileDialog.exe -t=m"

# [重要]
system "chcp 65001 >NUL"

def SubDialog()
	while true
		s1 = %x(#{CMD})
		break if s1.length == 0
		print s1
	end
end

SubDialog()

File: sample.py
#!python3
#coding:utf-8

import subprocess

CMD = ".\\iwm_DirFileDialog.exe -t=m"

# [重要]
subprocess.Popen("chcp 65001 >NUL", shell=True)

def SubDialog():
	while(True):
		child = subprocess.Popen(
			CMD,
			shell=True,
			stdout=subprocess.PIPE,
			stderr=subprocess.PIPE
		)
		stdout, stderr = child.communicate()
		if(len(stdout) == 0):
			break
		# [重要]
		print(stdout.decode("cp65001"), end="")

SubDialog()
